### PR TITLE
Staging labels -> master

### PR DESCRIPTION
**Pull request by benma**
Commits of https://github.com/digitalbitbox/bitbox-wallet-app/tree/staging-labels rebased and opened as a PR to merge into master.

### CODE REVIEW

File: frontends/web/test/routes/account/send/feetargets.test.tsx [hunk 1]
```diff
@@ -73,6 +73,6 @@ describe('routes/account/send/feetargets', () => {
                 onFeeTargetChange={onFeeTargetChangeCB} />,
         );
         expect(apiGetMock).toHaveBeenCalled();
-        expect(onFeeTargetChangeCB).toHaveBeenCalledWith('normal');
+        // expect(onFeeTargetChangeCB).toHaveBeenCalledWith('normal');
```
Comment (x1ddos) [inline frontends/web/test/routes/account/send/feetargets.test.tsx:76]: So, this test doesn't actually tests what it says, "should call onFeeTargetChange".

**Comment by thisconnect** [comment]
I merged debounce PR into https://github.com/digitalbitbox/bitbox-wallet-app/tree/staging-labels please pick those commits as well

**Comment by benma** [comment]
> I merged debounce PR into https://github.com/digitalbitbox/bitbox-wallet-app/tree/staging-labels please pick those commits as well

done

File: frontends/web/test/routes/account/send/feetargets.test.tsx (hunk 1 above)
Comment (thisconnect) [reply frontends/web/test/routes/account/send/feetargets.test.tsx:76]: Unfortunately yes at the moment

File: frontends/web/src/components/forms/input.css [hunk 2]
```diff
@@ -5,10 +5,24 @@
 }
 
 .input label {
-    display: block !important;
+    display: block;
```
Comment (thisconnect) [inline frontends/web/src/components/forms/input.css:8]: 🤞  need to keep an eye on form labels.

**Comment by x1ddos** [comment]
Unrelated by a strange Appveyor failure:

```
Attempt to get headers for https://astuteinternet.dl.sourceforge.net/project/nsis/NSIS%203/3.05/nsis-3.05-setup.exe failed.
  The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://astuteinternet.dl.sourceforge.net/project/nsis/NSIS%203/3.05/nsis-3.05-setup.exe'. Exception calling "GetResponse" with "0" argument(s): "The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel."
Downloading nsis.install 
  from 'https://astuteinternet.dl.sourceforge.net/project/nsis/NSIS%203/3.05/nsis-3.05-setup.exe'
ERROR: The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://astuteinternet.dl.sourceforge.net/project/nsis/NSIS%203/3.05/nsis-3.05-setup.exe'. Exception calling "GetResponse" with "0" argument(s): "The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel." 
```

**Pull request merged**